pandas/tests/arrays/test_datetimelike.py

```diff
@@ -45,6 +45,19 @@ def datetime_index(request):
     return pi
 
 
+@pytest.fixture
+def timedelta_index(request):
```

Comment:
eventually promote these to conftest

Reply:
Agreed.  For now this is a pretty bare-bones version to get the ball rolling.